packages/design-tokens/src/utils.ts

```diff
@@ -141,38 +164,71 @@ export const makeCSSVariableTheme = (
  * ```
  * {
  *    ...
- *    "--kz-var-theme-key": "zen",
- *    "--kz-var-color-wisteria-800": "#3537a4"
+ *    "--theme-key": "zen",
+ *    "--color-wisteria-800": "#3537a4"
  *    ...
  * }
  * ```
  */
 export const makeCSSVariablesOfTheme = (theme: Theme) =>
-  flattenObjectToCSSVariables({ [cssVariableThemeNamespace]: theme })
+  flattenObjectToCSSVariables({
+    ...theme,
+    [deprecatedKzVarNamespace]: theme,
+  })
 
 /**
- * Use this to generate an object containing `${key}: value`, `${key}-default: value`, and `${key}-rgb-params: r, g, b` if the value is a color.
- * This is for augmenting a CSS variable to support our solution to regression issues with using CSS variables instead of concrete values.
+ * Use this to generate an object containing `${key}: value`, `${key}-rgb: r, g, b`, and/or `${key}-id: --color-blah-XXX`.
+ * This is for adding extra neighbouring properties to a theme.
+ * For example:
+ *  Input:
+ *    path: [color, purple, 100]
+ *    key: 100
+ *    value: #f0f1f4
+ *    printValue: (path, v) => `var(--some-key, ${v})`
+ *    options: {augmentWithRgbParams: true, augmentWithId: true}
+ *
+ *  Output: {
+ *    "100": "var(--some-key, #f0f1f4)",
+ *    "100-rgb": "var(--some-key, 240, 241, 244)",
+ *    "100-id": "--color-purple-100"
+ *  }
  */
-export const augmentCssVariable = (
+export const augmentThemeKeyValue = (
```

Comment:
I just realised this is a breaking change...but there is a 99.999% chance nothing outside of this repo is using it lol